data/examples/shaders/source/shadertoy-MlKSWm.effect

```diff
@@ -0,0 +1,362 @@
+	vec4 h = 1.0 - abs(x) - abs(y);
+
+	vec4 b0 = vec4(x.xy, y.xy);
+	vec4 b1 = vec4(x.zw, y.zw);
+
+	//vec4 s0 = vec4(lessThan(b0,0.0))*2.0 - 1.0;
+	//vec4 s1 = vec4(lessThan(b1,0.0))*2.0 - 1.0;
+	vec4 s0 = floor(b0) * 2.0 + 1.0;
+	vec4 s1 = floor(b1) * 2.0 + 1.0;
+	vec4 sh = -step(h, vec4(0.0, 0.0, 0.0, 0.0));
+
+	vec4 a0 = b0.xzyw + s0.xzyw * sh.xxyy;
+	vec4 a1 = b1.xzyw + s1.xzyw * sh.zzww;
+
+	vec3 p0 = vec3(a0.xy, h.x);
+	vec3 p1 = vec3(a0.zw, h.y);
+	vec3 p2 = vec3(a1.xy, h.z);
+	vec3 p3 = vec3(a1.zw, h.w);
+
+//Normalise gradients
+	//vec4 norm = taylorInvSqrt(vec4(dot(p0,p0), dot(p1,p1), dot(p2, p2), dot(p3,p3)));
+	vec4 norm = rsqrt(vec4(dot(p0, p0), dot(p1, p1), dot(p2, p2), dot(p3, p3)));
+	p0 *= norm.x;
+	p1 *= norm.y;
+	p2 *= norm.z;
+	p3 *= norm.w;
+
+// Mix final noise value
+	vec4 m = max(0.6 - vec4(dot(x0, x0), dot(x1, x1), dot(x2, x2), dot(x3, x3)), 0.0);
+	m = m * m;
+	return 42.0 * dot(m * m, vec4(dot(p0, x0), dot(p1, x1), dot(p2, x2), dot(p3, x3)));
+}
+
+//////////////////////////////////////////////////////////////
+
+// PRNG
+// From https://www.shadertoy.com/view/4djSRW
+float prng(in vec2 seed)
+{
+	seed = fract(seed * vec2(5.3983, 5.4427));
+	seed += dot(seed.yx, seed.xy + vec2(21.5351, 14.3137));
+	return fract(seed.x * seed.y * 95.4337);
+}
+
+//////////////////////////////////////////////////////////////
+
+float noiseStack(vec3 pos, int octaves, float falloff)
+{
+	float noise = snoise(vec3(pos));
+	float off = 1.0;
+	if (octaves > 1)
+	{
+		pos *= 2.0;
+		off *= falloff;
+		noise = (1.0 - off) * noise + off * snoise(vec3(pos));
+	}
+	if (octaves > 2)
+	{
+		pos *= 2.0;
+		off *= falloff;
+		noise = (1.0 - off) * noise + off * snoise(vec3(pos));
+	}
+	if (octaves > 3)
+	{
+		pos *= 2.0;
+		off *= falloff;
+		noise = (1.0 - off) * noise + off * snoise(vec3(pos));
+	}
+	return (1.0 + noise) / 2.0;
+}
+
+vec2 noiseStackUV(vec3 pos, int octaves, float falloff, float diff)
+{
+	float displaceA = noiseStack(pos, octaves, falloff);
+	float displaceB = noiseStack(pos + vec3(3984.293, 423.21, 5235.19), octaves, falloff);
+	return vec2(displaceA, displaceB);
+}
+
+void mainImage(out vec4 fragColor, in vec2 fragCoord)
+{
+	vec2 resolution = iResolution.xy;
+	if (Burner)
+	{
+		resolution = -1.0 + 2.0 * (fragCoord.xy) / 1; //iResolution.xy;
+	}
+	resolution.x = mul(resolution.x, 1 / 1);
+	float time = iTime;
+	vec2 drag = iMouse().xy;
+	vec2 offset = iMouse().xy;
+	//
+	float xpart = fragCoord.x / resolution.x;
+	float ypart = fragCoord.y / resolution.y;
+	//
+	
+	float ypartClip = fragCoord.y / (clip * 75.0);
+	float ypartClippedFalloff = clamp(2.0 - ypartClip, 0.0, 1.0);
+	float ypartClipped = min(ypartClip, 1.0);
+	float ypartClippedn = (1 - ypartClipped);
+	//
+	float xfuel = pow(1.0 - abs(2.0 * xpart - 1.0), 0.5); //pow(1.0-abs(2.0*xpart-1.0),0.5);
+	//
+	float timeSpeed = 0.5;
+	float realTime = -1.0 * timeSpeed * time;
+	//
+	vec2 coordScaled = -1 * flameTongueSize * fragCoord - 0.02 * offset;
```

Comment:
This line is vastly different from the original. I don't know why Flame Tongue Size is negated here for no apparent reason.